### PR TITLE
hw-mgmt: scripts: Support labels on XDR platforms

### PR DESCRIPTION
**Pull request by ciju-nvidia**
This is the initial patches for supporting labels creation on XDR platforms (QM3400 & QM3000). The labels archive need to be created as part of the bring up and stored in /etc/hw-management-sensors/.

### CODE REVIEW

File: usr/etc/hw-management-sensors/qm3000_sensors_labels.json [hunk 1]
```diff
@@ -0,0 +1,237 @@
+{
+	"name": "qm3000",
+	"labels_qm3400_rev1_array" : {
```
Comment (sholeksandr) [inline usr/etc/hw-management-sensors/qm3000_sensors_labels.json:3]: If you using label name /format labels_${sku}_rev1_array, then it should be "labels_HI157_rev1_array" or "labels_HI15*_rev1_array".
On new systems SKU format HI{XXX}

File: usr/etc/hw-management-sensors/qm3400_sensors_labels.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
+{
+	"name": "qm3400",
+	"labels_qm3400_rev1_array" : {
```
Comment (sholeksandr) [inline usr/etc/hw-management-sensors/qm3400_sensors_labels.json:3]: If you using label name /format labels_${sku}_rev1_array, then it should be "labels_HI157_rev1_array" or "labels_HI15*_rev1_array".
On new systems SKU format HI{XXX}

File: usr/usr/bin/hw-management-labels-maker.sh [hunk 3]
```diff
@@ -160,7 +160,7 @@ make_labels()
 			mkdir -p "$label_dir"
 		fi
 		ln -sf "$attr_full_name" "$label_dir/$attr_file"
-		scale=$(hw_management_parse_labels.py --get_value --label "labels_scale_mqm9700_rev1_array" --key "$key")
```
Comment (sholeksandr) [inline usr/usr/bin/hw-management-labels-maker.sh:163]: if you are using sku in name, don't forget to remane in mqm9700'_labels.json: "labels_mqm9700_rev1_array" => "labels_HI130_rev1_array"

**Comment by ciju-nvidia** [comment]
RERUN

**Review by sholeksandr — APPROVED** [review]

**Pull request merged**